xml/System.Reflection.Emit/OpCodes.xml

```diff
@@ -6830,6 +6830,7 @@
  The following <xref:System.Reflection.Emit.ILGenerator.Emit%2A> method overload can use the `ldc.i4.s` opcode:  
```

Comment:
```suggestion
  The following <xref:System.Reflection.Emit.ILGenerator.Emit%2A> method overloads can use the `ldc.i4.s` opcode:  
```
If there are two of them, should be "overloads".

Reply:
Right, thanks for pointing it out!
Fixed in https://github.com/dotnet/dotnet-api-docs/pull/3216/commits/578d83b439e4ccb7eaf979d0a1769f3c4ef63695.